README.md

```diff
@@ -38,6 +38,9 @@
 
 ### Install script
 
+*Note: the instructions below will not work if you changed your shell from bash
+to [fish](https://fishshell.com/)  or some other shell which does not read the `.bashrc` file. If this is the case, please run `chsh -s /bin/bash`, log out and then log in again.*
```

Comment:
fish won't work regardless, because it's not POSIX compliant - this is something a fish user needs to just know.

This doesn't seem like a super common edge case to be near the top of the readme; perhaps a section in "troubleshooting" that talks about the problem?

Reply:
**Summary:**
I tested this on my Ubuntu 17.04 and you are right. :+1: I'm sorry for wasting your time. The second bold "Note" [here][2] (in the README.md of the nvm repository) talks about how fish and nvm work together, so I think this pull request can be closed.

**More details:**
I ran `chsh -s /usr/bin/fish`, logged out and logged in, and the `nvm` command was not found. After I ran `chsh -s /bin/bash`, logged out and logged in again, I also tried putting the line `exec fish` at the end of my `.bashrc` (as suggested [here][1]) and after this I opened a new terminal, the nvm command was still not found. If I removed that line from `.bashrc` and tried running nvm from bash, `nvm` worked. I later found [here][3] that `nvm` is not a script file or an executable, but a shell function.

  [1]: https://wiki.archlinux.org/index.php/Fish#Modify_.bashrc_to_drop_into_fish
  [2]: https://github.com/creationix/nvm#important-notes
  [3]: https://github.com/creationix/nvm#verify-installation